packages/template/vite/tmpl/forge.config.js

```diff
@@ -34,17 +34,19 @@ module.exports = {
           {
             // `entry` is just an alias for `build.lib.entry` in the corresponding file of `config`.
             entry: 'src/main.js',
-            config: 'vite.main.config.mjs',
+            config: 'vite.main.config.js',
+            target: 'main',
           },
           {
             entry: 'src/preload.js',
-            config: 'vite.preload.config.mjs',
+            config: 'vite.preload.config.js',
+            target: 'preload',
           },
         ],
         renderer: [
           {
             name: 'main_window',
-            config: 'vite.renderer.config.mjs',
+            config: 'vite.renderer.config.js',
```

Comment:
You switched from ESM to CJS here, can I understand this PR as a breaking change to the vite template 🤔? If users upgrade the vite template, they will need to convert their previous ESM code to CJS code.

Reply:
Vite will use esbuild to pre-build any config files with suffixes, such as '.ts', '.mts', '.cts', '.js', '.mjs', '.cjs', and they have no essential difference (and will not be affected by `"type": "module | commonjs"`).
Users only need to modify the config in **forge.config.js**.
I just want to give the decision back to the user, and the user should decide whether to enable ESM through "type": "module". Of course, this does not bring any benefits, I can revert it.